thirdeye-spi/src/main/java/ai/startree/thirdeye/spi/Constants.java

```diff
@@ -118,4 +118,9 @@ enum MonitorType {
     UPDATE,
     EXPIRE
   }
+
+  Integer LOW_METRIC_TIMEOUT = 1;
+  Integer MEDIUM_METRIC_TIMEOUT = 5;
+  Integer HIGH_METRIC_TIMEOUT = 15;
+  TimeUnit METRIC_TIMEOUT_UNIT = TimeUnit.MINUTES;
```

Comment:
Anyways we need to maintain the central `METRIC_TIMEOUT_UNIT` so created just an integer instead of a Duration object, will change that.
Yeah we can have single cache value, was just trying to have fresh metrics where we can afford. Few metrics such as `detectionTaskLatencyInMillis` is very costly hence kept a high timeout for that. Let me know if its too much complexity and we can default all to 15 mins.

Reply:
As per the discussion defaulting all timeouts to 10 mins and using Duration object to denote it.